### PR TITLE
Update doc, add missing parameter

### PR DESCRIPTION
**Pull request by fwolfst**

### CODE REVIEW

File: guides/testing/helpers.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ Additionally, it accepts some keyword arguments:
 You can use {{ "Testing::Helpers#with_resolution_context" | api_doc }} to use the same type, runtime object, and GraphQL context for multiple field resolutions. For example:
 
 ```ruby
-with_resolution_context(type: "Post", object: example_post, context: { current_user: author }) do |rc|
+with_resolution_context(YourSchema, type: "Post", object: example_post, context: { current_user: author }) do |rc|
```
Comment (rmosolgo) [inline guides/testing/helpers.md:52]:
```suggestion
# Assuming `include GraphQL::Testing::Helpers.for(MySchema)`
# was used above ...
with_resolution_context(type: "Post", object: example_post, context: { current_user: author }) do |rc|
```

**Review by rmosolgo — COMMENTED** [review]
Hey, thanks for suggesting a clarification here. It _should_ work as written if the schema-based helper module was included as documented above on the page. 

How about adding a note about that instead? I suggested one possible approach.

The helper module reduces some repetition in these calls so I'd like to recommend it to readers.

**Comment by fwolfst** [comment]
> Hey, thanks for suggesting a clarification here. It _should_ work as written if the schema-based helper module was included as documented above on the page.
> 
> How about adding a note about that instead? I suggested one possible approach.
> 
> The helper module reduces some repetition in these calls so I'd like to recommend it to readers.

Right. Like this, or even not as a comment, but as a statment (I tend to overlook comments ;) ).

**Comment by rmosolgo** [comment]
Thanks for the improvement!

**Pull request merged**